src/tests/IntegrationTests.cpp

```diff
@@ -1,22 +1,5 @@
-#include "CLIOptions.h"
-#include "MainOpt.h"
-#include "roundtrip.h"
-#include <CLI/CLI.hpp>
-#include <gtest/gtest.h>
-#include <librdkafka/rdkafka.h>
-
 int main(int argc, char **argv) {
-  ::testing::InitGoogleTest(&argc, argv);
-  CLI::App App{""};
-  auto Options = std::unique_ptr<MainOpt>(new MainOpt());
-  setCLIOptions(App, *Options);
-
-  CLI11_PARSE(App, argc, argv);
-  setup_logger_from_options(*Options);
-  SetTestOptions(Options.get());
-
-  return RUN_ALL_TESTS();
+  // Should be no longer needed.
```

Comment:
Agreed, any reason not to remove this file?

Reply:
:+1: